docker/Dockerfile

```diff
@@ -16,6 +16,9 @@ FROM ghcr.io/openzipkin/java:11.0.9_p11 as install
 
 WORKDIR /code
 COPY --from=scratch /code .
+# If we've seeded m2repository, copy it into the repository of our install image.
+# This prevents uncacheable downloads that occur during Docker build.
+RUN test -d /code/m2repository && cp -r /code/m2repository/* ~/.m2/repository/
```

Comment:
Is it not possible to point maven at `/code/m2repository` without copying?

Reply:
yeah I think so. I already monkeyed with the scripts to have them use mvnw so it should work.

Reply:
🤞

Reply:
it wasn't so easy, but it is better in general now I think.